deploy/storage/bigtable.version

```diff
@@ -1 +1 @@
-borgcron_2021_02_05_01_04_21
\ No newline at end of file
+borgcron_2021_02_10_18_03_48
```

Comment:
lint caught this:), should not have newline

Reply:
gah :(

if i change my vim settings to not have newline google3 complains...

Reply:
Ok, I should just do this:

printf "<VERSION>" > deploy/storage/bigtable.version
printf "<VERSION>" > deploy/storage/bigquery.version

Reply:
Can try vscode for github code development. It's very convenient with all the plugins in place..

Reply:
Useful to know.  But since most of my code dev is still in g3, prefer an editor that works for both... so, stuck to simple living with vim in a terminal :)